Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -60,14 +60,11 @@ During this stage, the release manager has the following tasks:
 *It starts after the App Center build has been delivered and it can take several cycles*
 
 1. Testers will then begin their work against the build you just created.
-1. Any hotfix should target the `develop` branch first.
-1. These PRs need to be reviewed by the relevant squad or platform QA (Bibin Paul) & one of the release engineers assigned to the release. This is to ensure visibility of changes being requested to be made, and to ensure the correct builds are available for validation. The exception being that if an issue is not reproducible on `develop` then we can merge directly with `release`. Still, it should be fully understandable why it is not reproducible to help ensure there is no regression for the next release.
+1. Any hotfix should target the release branch first.
+1. These PRs need to be reviewed by the relevant squad or platform QA (Bibin Paul) & one of the release engineers assigned to the release. This is to ensure visibility of changes being requested to be made, and to ensure the correct builds are available for validation.
```

Comment:
Being the process we should just reference the role and not the name. If we want to keep the name then we can have a separate section where we can assign people to the roles (*release manager, platform QA, etc...)
```suggestion
1. These PRs need to be reviewed by the relevant squad or **platform QA** & one of the release engineers assigned to the release. This is to ensure visibility of changes being requested to be made, and to ensure the correct builds are available for validation.
```